docs/OMS-MySQL-Instructions.md

```diff
@@ -0,0 +1,88 @@
+# MySQL Server and Log Monitoring Solution for Operations Management Suite
+
+1. Setup a Linux(Ubuntu/Redhat) machine and install [MySQL](http://dev.mysql.com/doc/refman/5.7/en/installing.html).
+
+2. Download and Install [OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux) on the machine. 
+
+3. Configure MySQL to generate slow, error and general logs.
```

Comment:
Let's point them to MySQL documentation on setting up these logs: http://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html